style-guide.md

```diff
@@ -0,0 +1,111 @@
+# Dockstore Docs Style Guide
+
+This style guide is based upon [Google's style guide](https://developers.google.com/style/) but contains several sections specific to Dockstore itself. It is not exhaustive nor authoritative; this document is designed as a quick reference guide.
+
+## Accessibility
+See also: https://developers.google.com/style/accessibility
+
+* Always set descriptive alt text for images, except for decorative images
+ 	* If the image contains important text, repeat that text in the alt text
+	 	* Although it is tricky due to how documentation websites tend to use images to clarify written text,  try to avoid having alt text repeat what is in the body text
+	 	* Do not start with "image of" or "picture of" but do describe the type of image, like:
+	 		* "Headshot of David Haussler smiling"
+		 	* "Screenshot of the UI of dockstore.org/my-tools with the Refresh Organization button circled"
+		 	* "Animation of a tabby cat jumping three feet into the air upon noticing a cucumber"
+* Keep in mind ~5% of the population is red-green colorblind
+	* 	When color schemes convey non-essential meaning, avoid using color schemes that would be inaccessible to red-green colorblind users
+	* 	When color schemes convey essential meaning (vital UI elements, etc) they should be functional in grayscale
+* Do not use things that would be a nightmare for screen readers
+ 	* Avoid using non-ASCII symbols unless you are writing something meaningful in another language
+ 		* For example, do not use symbols from U+1D400 𝒔𝒖𝒄𝒉 𝒂𝒔 𝒕𝒉𝒆𝒔𝒆 as a replacement for ***actual bold and italic formatting***
+		* Using actual bold and italics formatting is acceptable, but bear in mind screen readers may not declare something as being bold or italic
+
+## Assumptions about a user's system
+* If talking about a program that does not support certain operating systems, make it clear which systems are supported relatively early in documentation introducing that program
+	* If possible, give an alternative for unsupported OSs
+	* "The Dockstore CLI is only supported on Unix-like operating systems, including Linux and Mac. Windows users may wish to..."
+* Do not assume users are or are not using an HPC
```

Comment:
Following on from comment, there's a lot of "do not" in here. 
For  hpc, what users know about hpc, Apple Silicon Mac, different versions of Bash, how programs function on HPC, etc. 

Might be simpler and shorter just to say what you "should" do, write instructions that note what OS and situation you're working in and note exceptions that you're aware of which may include (examples above)

Reply:
How about this? https://github.com/dockstore/dockstore-documentation/pull/197/commits/eaa9fc5d380d201b893958776de549945cd38171

Reply:
I think so, I find that a little easier to follow